### ISSUE TITLE
“Next →” button in Page Topics/Messages don't work

### ISSUE THREAD

**Issue by xayjin** [provectus/kafka-ui#3622]
**Describe the bug** (Actual behavior)

“Next →” button in Page Topics/Messages don't work. 

**Expected behavior**

It remains the Offset 0-99 after clicked, which is supposed to be next Offest 100-199.

**Set up**

os version: debian 11 bullseye
java: openjdk-17
chrome: 110.0.5481.178(x64)
shell script is:
```
#!/bin/bash
export some simple env, such as server and jaas configure
java -jar kafka-ui-api-v0.6.1.jar
```

**Steps to Reproduce**

1. simple run the script
```
#!/bin/bash
export some simple env, such as server and jaas configure
java -jar kafka-ui-api-v0.6.1.jar
```

**Screenshots**

**Additional context**

“Next” button works well in release version kafka-ui-api-v0.4.0.jar. Don't work in v0.5.0 v0.6.0.

**Comment by xayjin** [comment]
v0.6.1 Next button background query is http://messages?filterQueryType=STRING_CONTAINS&attempt=0&limit=100&seekDirection=FORWARD&seekType=OFFSET&seekTo=0::200
v0.4.0 Next button background query is http://messages?filterQueryType=STRING_CONTAINS&attempt=1&limit=100&page=1&seekDirection=FORWARD&keySerde=String&valueSerde=String&seekType=BEGINNING
v0.4.0 query works well in v0.6.0.

**Comment by Haarolean** [comment]
Hi, thanks for raising the issue.

That's a duplicate of #3129 and is blocked by #3504.

Please follow & upvote #3129
